### PR TITLE
`part3_finding_point_of_interests.ipynb` guide fixed

### PR DESCRIPTION
**Pull request by tarunkukreja003**
`part3_finding_point_of_interests.ipynb` guide fixed.

Major changes:
       - changing from gis.map() to Map()
       - adding `spatialReference` to location for the Point geometry to be drawn on the Map
       - removed the section `Example of finding restaurants within a polygon drawn on map` since on_draw_end() function doesn't exist anymore
       - added updated map screenshots

### CODE REVIEW

**Review by nanaeaubry — APPROVED** [review]
Code looks good

**Review by jyaistMap — CHANGES_REQUESTED** [review]
submitted [pr #5](https://github.com/tarunkukreja003/arcgis-python-api-fork/pull/5) against your fork to correct some misaligned maps and remove links to tool that's no longer online

**Review by jyaistMap — APPROVED** [review]
- approved after pr merge and nbreview

**Pull request merged**
